ibm/service/power/resource_ibm_pi_workspace.go

```diff
@@ -121,6 +122,11 @@ func resourceIBMPIWorkspaceRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 	d.Set(PIWorkspaceName, controller.Name)
+	wsDetails := map[string]interface{}{
+		WorkspaceCreationDate: controller.CreatedAt,
+		WorkspaceCRN:          controller.TargetCRN,
+	}
+	d.Set(Attr_WorkspaceDetails, flex.Flatten(wsDetails))
```

Comment:
I do not see `Attr_WorkspaceDetails` defined in the resource schema. Did you run the tests to ensure the code works?
Also, please update the docs with the new attribute.

Reply:
Added

Reply:
I still don't see it in the schema.

Reply:
Added in schema